samza-azure/src/main/java/org/apache/samza/system/eventhub/EventHubConfig.java

```diff
@@ -36,15 +36,15 @@
 
   public static final String CONFIG_STREAM_LIST = "systems.%s.stream.list";
 
-  public static final String CONFIG_STREAM_NAMESPACE = "systems.%s.streams.%s.eventhubs.namespace";
+  public static final String CONFIG_STREAM_NAMESPACE = "streams.%s.eventhubs.namespace";
```

Comment:
Just to confirm, this is going to be an incompatible change, right? I need to take notes and remember to adapt to this in my work

Reply:
@lhaiesp Yes that is correct!

Reply:
@lhaiesp Yes that is correct!